_includes/manuals/nightly/2.1_quickstart_installation.md

```diff
@@ -64,14 +65,26 @@ sudo yum -y install https://yum.theforeman.org/releases/{{page.version}}/el7/x86
 {% endhighlight %}
 </div>
 
-<div class="quickstart_os quickstart_os_el7">
+<div class="quickstart_os quickstart_os_centos7">
   <p>Enable the EPEL (Extra Packages for Enterprise Linux), Foreman and SCL (Software Collections) repositories:</p>
 
 {% highlight bash %}
-sudo yum -y install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+sudo yum-config-manager --enable extras
+sudo yum -y install epel-release centos-release-scl-rh
 sudo yum -y install https://yum.theforeman.org/releases/{{page.version}}/el7/x86_64/foreman-release.rpm
-sudo yum -y install foreman-release-scl
 {% endhighlight %}
+
+</div>
+
+<div class="quickstart_os quickstart_os_other_el7">
+  <p>Enable the EPEL (Extra Packages for Enterprise Linux), Foreman and SCL (Software Collections) repositories:</p>
+
+{% highlight bash %}
+sudo yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+sudo yum -y install https://yum.theforeman.org/releases/{{page.version}}/el7/x86_64/foreman-release.rpm
+sudo yum -y install https://mirror.centos.org/centos/7/extras/x86_64/Packages/centos-release-scl-rh-2-3.el7.centos.noarch.rpm
```

Comment:
In 3.3.1 we say that Oracle Linux should use [Software Collection Library documentation](http://docs.oracle.com/cd/E37670_01/E59096/html/index.html). For SL there's [softwarecollections on ftp.scientificlinux.org](http://ftp.scientificlinux.org/linux/scientific/6x/external_products/softwarecollections/). Note that links to `6x` already.

Should we leave out the exact instruction to install the release part and just leave it to those links? This feels inconsistent now.

Reply:
We mention it up above, that feels sufficient.

Reply:
Mind updating the Scientific Linux link to 7x? Thankfully we've dropped EL6 support a long time ago. Perhaps this indicates how little SL7 users we have that nobody complained about this. Since there will be no SL8, this will be simplified over time when we drop EL7 support.